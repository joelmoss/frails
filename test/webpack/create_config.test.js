const createConfig = require("../../package/webpack/create_config");

describe("createConfig", () => {
  test("returns object", () => {
    expect(createConfig()).toMatchSnapshot();
  });
});
