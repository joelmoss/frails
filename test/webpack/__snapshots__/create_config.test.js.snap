// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createConfig returns object 1`] = `
Object {
  "entry": "./app/assets/application.js",
  "mode": "development",
  "output": Object {
    "path": "/Users/joelmoss/dev/frails/test/dummy/public/assets",
    "publicPath": "/assets/",
  },
}
`;
